.github/workflows/publish.yml

```diff
@@ -7,15 +7,16 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 0 # for lerna
+          fetch-depth: 0
```

Comment:
this is still needed?

Reply:
it should at least make cloning much faster. There have been some giant files historicaly in the repo